tasks/install.yml

```diff
@@ -1,4 +1,16 @@
 ---
+- name: Create ClickHouse group
+  group:
+    name: "{{ clickhouse_group }}"
+    state: present
+
+- name: Create ClickHouse user
```

Comment:
are they created also by the debian package for CH?

Reply:
@pablo2go , good point, actually this is another time bomb... Let's remove the user/group creation and just set the vars internally.